ds3-autogen-api/src/main/java/com/spectralogic/ds3autogen/api/models/Ds3Type.java

```diff
@@ -101,4 +104,8 @@ public boolean equals(final Object obj) {
     public static boolean isEmpty(final Collection<?> collection) {
```

Comment:
Correct, we have it in the ds3-autogen-utils package. If I use that version of the function, then it changes all package dependencies (otherwise there are circular dependencies). Do you want me to make that change?